Game/Source/GDKTestGyms/Private/UserExperienceComponent.cpp

```diff
@@ -96,6 +100,16 @@ float UUserExperienceComponent::CalculateAverageUpdateTimeDelta() const
 	return Avg;
 }
 
+void UUserExperienceComponent::CheckPosition()
+{
+	FVector CurrPos = GetOwner()->GetActorLocation();
+	if (PreviousPos == CurrPos)
+	{
+		UE_LOG(LogTemp, Warning, TEXT("UserExperience failure: Client position not updating (%s). Vel %s"), *PreviousPos.ToString(), *GetOwner()->GetVelocity().ToString());
```

Comment:
Similarly, will we fail NFRs based on the presence of this log?

Reply:
Same as above